packages/chrome/src/elements/header/HeaderItem.tsx

```diff
@@ -19,6 +19,7 @@ interface IHeadItemProps
   extends IStyledBaseHeaderItemProps,
     IStyledLogoHeaderItemProps,
     HTMLAttributes<HTMLElement> {
+  /** Applies a logo to the header */
   hasLogo?: boolean;
```

Comment:
This prop does not add a logo to the header. Rather it indicates that the header *item* contains a product logo.

Reply:
Updated in latest push.